.devcontainer/devcontainer.json

```diff
@@ -8,18 +8,20 @@
       "extensions": [
         "ms-python.python",
         "github.vscode-pull-request-github",
-        "ryanluker.vscode-coverage-gutters",
```

Comment:
can you explain these changes

Reply:
This will install extensions that seem to be used by the project.